<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-12-31T13:34:50 IST" name="Default suite" finished-at="2023-12-31T13:36:18 IST" duration-ms="87969">
    <groups>
    </groups>
    <test started-at="2023-12-31T13:34:50 IST" name="Default test" finished-at="2023-12-31T13:36:18 IST" duration-ms="87969">
      <class name="Testcases.TC2_SignInAccountPage2_Excelread">
        <test-method is-config="true" signature="setup()[pri:0, instance:Testcases.TC2_SignInAccountPage2_Excelread@33d512c1]" started-at="2023-12-31T13:34:50 IST" name="setup" finished-at="2023-12-31T13:36:09 IST" duration-ms="78993" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="VerifyLogin(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:Testcases.TC2_SignInAccountPage2_Excelread@33d512c1]" started-at="2023-12-31T13:36:09 IST" name="VerifyLogin" data-provider="LoginDataProvider" finished-at="2023-12-31T13:36:18 IST" duration-ms="8854" status="FAIL">
          <exception class="org.testng.internal.reflect.MethodMatcherException">
            <message>
              <![CDATA[
[public void Testcases.TC2_SignInAccountPage2_Excelread.VerifyLogin(java.lang.String,java.lang.String,java.lang.String) throws java.io.IOException] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: VerifyLogin([java.lang.String arg0, java.lang.String arg1, java.lang.String arg2])
Arguments: [(java.lang.String) vaishalimore0294@gmail.com,(java.lang.String) Vaishali@0294,(java.lang.String) Vaishali More,(java.lang.String)  ]]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.internal.reflect.MethodMatcherException: 
[public void Testcases.TC2_SignInAccountPage2_Excelread.VerifyLogin(java.lang.String,java.lang.String,java.lang.String) throws java.io.IOException] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: VerifyLogin([java.lang.String arg0, java.lang.String arg1, java.lang.String arg2])
Arguments: [(java.lang.String) vaishalimore0294@gmail.com,(java.lang.String) Vaishali@0294,(java.lang.String) Vaishali More,(java.lang.String)  ]
at org.testng.internal.reflect.DataProviderMethodMatcher.getConformingArguments(DataProviderMethodMatcher.java:46)
at org.testng.internal.Parameters.injectParameters(Parameters.java:909)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:37)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.internal.reflect.MethodMatcherException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyLogin -->
      </class> <!-- Testcases.TC2_SignInAccountPage2_Excelread -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
